CHANGELOG.md

```diff
@@ -3,9 +3,14 @@ cross-compilation issue, but will return in v0.13.0.
 
 # Main (unreleased)
 
+# 0.14.0-rc.3 (2021-04-15)
```

Comment:
Are we doing an rc 3 today?

Reply:
Would like to so that everyone gets the auto start if they download Grafana Agent.